Some homepage styling issue fixes

bosschaert opened the pull request:
Some fixes for the homepage styling, addresses:
* banner shade
* switching section styling
* promotion header and arrow location
* added `-webkit-font-smoothing: antialiased;` to the root style to make the font look better (this setting is also in the original website)

It does not yet address the navbar popup styling issues.

Fix #87

Test URLs:
- Before: https://main--nedbank--hlxsites.hlx.page/content/nedbank/za/en/personal/home
- After: https://issue-87--nedbank--hlxsites.hlx.page/content/nedbank/za/en/personal/home

bosschaert commented:
I created this as a draft PR because I'm not 100% sure about some things yet.
Because the font is slightly different from the original things cannot be made to look 100% identical.

Also some text pieces have no style class associated with it, therefore I used just the`#id` syntax to refer to them. Is that considered ok? 

Any other suggestions?

vdua commented on blocks/cards/cards.less:
```diff
@@ -855,6 +855,37 @@ autocomment();
+
+  + p {
+    font-size: 16px;
+    line-height: 24px;
```
would suggest to use less variables. As an example for font-size https://github.com/hlxsites/nedbank/blob/main/styles/variables.less#L84

bosschaert replied:
Good point, thanks @vdua ! I will update the PR with that.

sdmcraft commented on blocks/cards/cards.css:
```diff
@@ -596,6 +596,31 @@
     display: flex;
   }
 }
+#switch-to-nedbank-hassle-free {
```
Using this kind of selector could be problematic. If the author changes the text in the word doc, this selector would no longer work. Maybe use another selector to  achieve the same. Also I see that this is a `h3` . Does it need to be styled differently from the  global `h3` styles?

bosschaert commented:
Thanks for the feedback every @vdua and @sdmcraft - I think I addressed your concerns with these changes now.

The dropdown menu styling issue is not yet addressed. I'll look at that separately.
